docs/interface.md

```diff
@@ -0,0 +1,56 @@
+# The Interface
+
+All interactions with the Exercism website are handled automatically. Test runners have the single responsibility of taking a solution, running its tests and returning a standardised output.
```

Comment:
Maybe use "all" instead of "its" to emphasize that the test runner has to do something to also run the skipped tests?

```suggestion
All interactions with the Exercism website are handled automatically. Test runners have the single responsibility of taking a solution, running all tests and returning a standardised output.
```